voice-client-javascript/assets/browser.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html>
```

Comment:
would it be possible to adapt this page to use the new index.html template design? Possibly the admin page as well?